tasks/javascript/easy/sum.js

```diff
@@ -2,3 +2,11 @@
 
 // TODO: Write a function that returns the sum of two numbers
 
+const num1 = 1;
+const num2 = 2;
+
+// add two numbers
+const sum = num1 + num2;
+
```

Comment:
Code looks good! However, the issue requested a function that returns a sum of two numbers so you could do that with arrow function:
```bash
const sum = (a, b) => a + b
const num1 = 1
const num2 = 2
console.log('The sum of ' + num1 + ' and ' + num2 + ' is ' + sum(num1, num2) + '.')
```
But it's fine, im still going to merge this. Thanks for contributing!